fsh-generated/resources/ValueSet-task-status.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
Commenting here to get an inline comment. 

Please update change log - related JIRA ticket: https://jira.hl7.org/browse/FHIR-47081

Reply:
It appears that 47801 is already mentioned in the change log file.